src/newrelic/synthetics.js

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2019 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+const request = require('request-promise-native');
+const fs = require('fs');
+const path = require('path');
+
+const MONITOR_FREQUENCY = 15;
+const MONITOR_STATUS = 'ENABLED';
+const MONITOR_THRESHOLD = 7;
+const MONITOR_LOCATIONS = [
```

Comment:
I don't know the full story behind this, but should we make this configurable (maybe in a separate PR). Should we reduce the interval to make up for the reduction in locations?

Reply:
Readded all AWS locations after confirming with New Relic.